ensembling/qra-ensemble-sum-of-states/update-sum-of-states-qra-ensemble.R

```diff
@@ -0,0 +1,59 @@
+
+library(magrittr); library(ggplot2)
+
+# load latest qra-ensemble
+state_qra <- readr::read_csv(file = here::here("ensembling",
+                                               "qra-state-ensemble",
+                                               "submission-files",
+                                               "latest-epiforecasts-ensemble1-qra.csv"))
```

Comment:
new name change (sorry!):

file = here::here("ensembling",
                                               "qra-state-ensemble",
                                               "submission-files",
                                               "latest.csv"))

Reply:
So this will be latest.csv across all models? Is there any change to the dated files that I need to make as well?